content/en/docs/zero-code/dotnet/_index.md

```diff
@@ -84,10 +84,16 @@ brew install coreutils
 
 ### Windows (PowerShell)
 
+{{% alert title="Note" color="warning" %}} PowerShell v6.0+ (pwsh) is not
+supported to install .NET zero-code instrumentation. The built-in version of
+Windows PowerShell must be used. {{% /alert %}}
+
 On Windows, use the PowerShell module as an Administrator:
```

Comment:
```suggestion
On Windows, use the PowerShell module as an Administrator.

{{% alert title="Version note" color="warning" %}}

PowerShell v5.1 is required. In particular, v6.0+ is not yet supported.

{{% /alert %}}
```

Reply:
I'll be OOO so I'll let @theletterf and @open-telemetry/dotnet-approvers follow up and merge when they see fit.